### PR TITLE
Release notes for 1.2

### PR DESCRIPTION
**Pull request by sjberman**
Add release notes for the upcoming 1.2 release of the controller.

Removed empty fields from previous release notes.

### CODE REVIEW

File: docs/RELEASE-NOTES.rst [hunk 1]
```diff
@@ -4,6 +4,22 @@ Release Notes for BIG-IP Controller for Kubernetes
 |release|
 ---------
 
+Added Functionality
+```````````````````
+
+* Watch all nodes by default; watch nodes with a user-specified label.
```
Comment (amudukutore) [inline docs/RELEASE-NOTES.rst]: How about:
Watch all nodes by default; watch a subset of node with a user-specified node label.

File: docs/RELEASE-NOTES.rst [hunk 2]
```diff
@@ -4,6 +4,22 @@ Release Notes for BIG-IP Controller for Kubernetes
...
+* Custom SSL Profiles can be loaded from Kubernetes Secrets via Ingress TLS.
```
Comment (amudukutore) [inline docs/RELEASE-NOTES.rst]: Are we loading (i.e., creating) these on the BIG-IP? If so, how about:
Creation of BIG-IP SSLProfiles from Kubernetes secrets via Ingress TLS.

File: docs/RELEASE-NOTES.rst [hunk 3]
```diff
@@ -4,6 +4,22 @@ Release Notes for BIG-IP Controller for Kubernetes
...
+* Creation of BIG-IP objects from OpenShift Route resources.
+  - This includes unsecured, Edge, Passthrough, and Re-encrypt Routes
+* Can properly configure http redirect rules on v11.6.1 BIG-IP systems
```
Comment (amudukutore) [inline docs/RELEASE-NOTES.rst]: Is this a bugfix?

Comment (sjberman) [reply docs/RELEASE-NOTES.rst]: Yes, should I include a Bug Fixes section?

**Comment by sjberman** [comment]
Just a note, the build fails at the moment because the docs link I included doesn't exist yet. It will exist once the OpenShift solution docs are live.

File: docs/RELEASE-NOTES.rst [hunk 4]
```diff
@@ -4,6 +4,25 @@ Release Notes for BIG-IP Controller for Kubernetes
...
+* Creation of BIG-IP SSL Profiles from Kubernetes Secrets via Ingress TLS.
+* Creation of BIG-IP objects from OpenShift Route resources.
+  - This includes unsecured, Edge, Passthrough, and Re-encrypt Routes
+
+* This is a feature complete upgrade from the OpenShift F5Router. See the documentation for details
+  on how to upgrade: http://clouddocs.f5.com/containers/v1/openshift/replace-f5-router.html
+
+Bug Fixes
+`````````
+* Can properly configure http redirect rules on v11.6.1 BIG-IP systems
```
Comment (amudukutore) [inline docs/RELEASE-NOTES.rst]: I am no grammar/voice expert but I would drop 'Can' and add a period at the end to keep it consistent with other bullet items.

File: docs/RELEASE-NOTES.rst [hunk 5]
```diff
@@ -4,6 +4,25 @@ Release Notes for BIG-IP Controller for Kubernetes
 |release|
 ---------
 
+Added Functionality
+```````````````````
+
+* Watch all nodes by default; watch a subset of nodes with a user-specified label.
+* Creation of BIG-IP SSL Profiles from Kubernetes Secrets via Ingress TLS.
```
Comment (jputrino) [inline docs/RELEASE-NOTES.rst]: Creation of --> Create

File: docs/RELEASE-NOTES.rst [hunk 6]
```diff
@@ -4,6 +4,25 @@ Release Notes for BIG-IP Controller for Kubernetes
...
+* Creation of BIG-IP objects from OpenShift Route resources.
```
Comment (jputrino) [inline docs/RELEASE-NOTES.rst]: Creation of --> Create

File: docs/RELEASE-NOTES.rst [hunk 7]
```diff
@@ -4,6 +4,25 @@ Release Notes for BIG-IP Controller for Kubernetes
...
+  - This includes unsecured, Edge, Passthrough, and Re-encrypt Routes.
```
Comment (jputrino) [inline docs/RELEASE-NOTES.rst]: either capitalize "unsecured" or make all 4 lowercase

File: docs/RELEASE-NOTES.rst [hunk 8]
```diff
@@ -4,6 +4,25 @@ Release Notes for BIG-IP Controller for Kubernetes
...
+
+* This is a feature complete upgrade from the OpenShift F5Router. See the documentation for details
```
Comment (jputrino) [inline docs/RELEASE-NOTES.rst]: `See [Replace the OpenShift F5 Router with the BIG-IP Controller](http://clouddocs.f5.com/containers/elbert/openshift/replace-f5-router.html) for more information. `

File: docs/RELEASE-NOTES.rst [hunk 9]
```diff
@@ -4,6 +4,26 @@ Release Notes for BIG-IP Controller for Kubernetes
...
+Added Functionality
+```````````````````
+
+* Watch all nodes by default; watch a subset of nodes with a user-specified label.
+* Create BIG-IP SSL Profiles from Kubernetes Secrets via Ingress TLS.
+* Create BIG-IP objects from OpenShift Route resources.
+  - This includes unsecured, edge, passthrough, and re-encrypt Routes.
+
+* This is a feature complete upgrade from the OpenShift F5Router. 
+  See `[Replace the OpenShift F5 Router with the BIG-IP Controller] <http://clouddocs.f5.com/containers/elbert/openshift/replace-f5-router.html>`_
```
Comment (amudukutore) [inline docs/RELEASE-NOTES.rst]: replace elbert with v1.

File: docs/RELEASE-NOTES.rst [hunk 10]
```diff
@@ -4,6 +4,26 @@ Release Notes for BIG-IP Controller for Kubernetes
...
+* Create BIG-IP objects from OpenShift Route resources.
+  - This includes unsecured, edge, passthrough, and re-encrypt Routes.
+
+* This is a feature complete upgrade from the OpenShift F5Router. 
+  See `[Replace the OpenShift F5 Router with the BIG-IP Controller] <http://clouddocs.f5.com/containers/elbert/openshift/replace-f5-router.html>`_ 
+  for more information.
+
+Bug Fixes
+`````````
+* Properly configures http redirect rules on v11.6.1 BIG-IP systems.
```
Comment (amudukutore) [inline docs/RELEASE-NOTES.rst]: configure instead of configures?

File: docs/RELEASE-NOTES.rst [hunk 11]
```diff
@@ -4,6 +4,26 @@ Release Notes for BIG-IP Controller for Kubernetes
...
+* Failed configurations for objects should not prevent future configurations from happening.
```
Comment (amudukutore) [inline docs/RELEASE-NOTES.rst]: If we are describing the bugfix, I would prefer 'do not' instead of 'should not'.

**Pull request merged**